### PR TITLE
Pass scraper suffix to warc2zim

### LINKED ISSUE #275: Pass scraper parameter to warc2zim
Once https://github.com/openzim/warc2zim/issues/156 is implemented, we have to pass `zimit` version to `warc2zim` as `--scraper` so that the final ZIM scraper metadata is set properly.

### PR DESCRIPTION
**Pull request by benoit74**
Fix #275

### CODE REVIEW

**Comment by benoit74** [comment]
I forced-push a small change since I forgot to update the CHANGELOG

File: src/zimit/zimit.py [hunk 1]
```diff
@@ -360,6 +360,22 @@ def run(raw_args):
 
     zimit_args, warc2zim_args = parser.parse_known_args(raw_args)
 
+    logger.info("Checking browsertrix-crawler version")
+    crawl_version_cmd = ["crawl", "--version"]
+    crawl = subprocess.run(crawl_version_cmd, check=False, capture_output=True)
+    if crawl.returncode:
```
Comment (rgaudin) [inline src/zimit/zimit.py:366]: What's the point of this? Just run with `check=True` and except on `CalledProcessError`.

File: src/zimit/zimit.py [hunk 2]
```diff
@@ -360,6 +360,22 @@ def run(raw_args):
...
+        raise subprocess.CalledProcessError(crawl.returncode, crawl_version_cmd)
+    else:
+        crawler_version = crawl.stdout.decode("utf-8").strip()
```
Comment (rgaudin) [inline src/zimit/zimit.py:369]: Again, use `text=True` so you don't have to decode manually 😉

File: tests-integration/integration.py [hunk 3]
```diff
@@ -20,11 +16,22 @@ def test_zim_main_page():
     """Main page specified, http://isago.rskg.org/, was a redirect to https
     Ensure main page is the redirected page"""
 
-    main_entry = get_zim_main_entry("/output/isago.zim")
+    main_entry = LibzimArchive("/output/isago.zim").main_entry
```
Comment (rgaudin) [inline tests-integration/integration.py:19]: Why using pylibzim's Archive for this?

**Review by rgaudin — CHANGES_REQUESTED** [review]

File: tests-integration/integration.py (hunk 3 above)
Comment (benoit74) [reply tests-integration/integration.py:19]: This is how it was done. I imagined there was good reasons for it. Should I understand there are none?

Comment (rgaudin) [reply tests-integration/integration.py]: Yes, probably dates from before we added scraperlib to zimit

**Review by rgaudin — APPROVED** [review]

**Pull request merged**